projects/05-fleet-management-api/README.md

```diff
@@ -62,121 +62,162 @@ aplicación.
 > Acá puedes ver una [lista de todos los objetivos de aprendizaje](../../learning-objectives/data.yml)
 > que contempla nuestra currícula.
 
-## 4. Criterios de aceptación mínimos del proyecto
+## 4. Criterios de aceptación del proyecto
 
-### Funcionalidades mínimas
+ La empresa que nos ha contratado ha instalado dispositivos GPS en sus taxis.
+ Estos dispositivos utilizan señales satelitales para determinar
+ con precisión las coordenadas geográficas del taxi.
+ Hasta el momento la información de ubicación se ha almacenado en archivos
+ de texto, sin embargo, no es óptimo consultar la información con esta opción.
 
-Tu aplicación como mínimo deberá tener un Front-end y Back-end que permita:
+Nuestra clienta requiere:
 
-* Visualizar en un mapa la ubicación más reciente de todos los vehículos
-* Visualizar en un mapa todas las ubicaciones de un vehículo en una fecha
-específica
-* Almacenar y consultar los 17 millones de ubicaciones de los 10 mil taxis
-que te entregaremos.
-* **Garantizar tiempos de respuesta mínimos para una adecuada experiencia de
-usuaria**.
+1. Cargar la información de los archivos de texto a una
+base de datos Postgresql.
+2. Desarrollar una API REST que permita consultar, mediante
+peticiones HTTP, la información almancenada en la base de datos.
 
-Recuerda escribir Historias de Usuario, crear un prototipo de baja y alta
-fidelidad, así como tests de usabilidad.
+### Definición del producto
 
-### Modularización del código
+El [_Product Owner_](https://www.youtube.com/watch?v=r2hU7MVIzxs&t=202s)
+nos presenta este _backlog_ que es el resultado de su trabajo con el clientx
+hasta hoy.
 
-Deberás definir la estructura de carpetas y archivos que consideres
-necesaria, tanto para el Front-end como para el Back-end. Puedes guiarte de
-las convenciones del framework elegido. Por ende, los tests y el setup
-necesario para ejecutarlos, serán hechos por ti.
+***
 
-### Pruebas unitarias
+#### [Historia de usuario 1] Cargar información a base de datos
 
-Deberás incluir pruebas unitarias para el Front-end y el Back-end de la
-aplicación que desarrolles.
-Te invitamos a escribir casos de prueba para las principales funcionalidades.
+Yo como _desarrolladora_ requiero cargar la información, almacenada
+hasta ahora en archivos de texto, en una base de datos Postgresql
+para facilitar su consulta y análisis.
 
-## 5. Stack de tecnologías
+##### Criterios de aceptación
 
-### Arquitectura de la aplicación
+* Desarrollar una Interfaz de Línea de Comando (CLI) que
+cargue la información de los archivos de texto a la base
+de datos
 
-La arquitectura que te recomendamos para tu aplicación es la siguiente:
+##### Definición de terminado
 
-![Arquitectura](./docs/architecture.jpg "Arquitectura recomendada")
+* El código de la Interfaz de Línea de Comando (CLI) debe
+recibir _code review_ de al menos una compañera.
```

Comment:
Necesitamos poner algunos criterios sobre como hacer esto no?

Reply:
Especifique la siguiente firma para el CLI

upload-gps-data <path-to-files> 
--type=taxis|trajectories
--dbname=<dbname>
--host=<hostname>
--port=<port>
--username=<username>

Reply:
va! te parece si lo proponemos de frente con java o node. Por ejemplo
`java UploadGPSData.java`
o
`node upload-gps-data.js`

si no, vamos a tener q meternos con bash scripting y las diferencias en windows y linux